libnavigation-ui/src/main/java/com/mapbox/navigation/ui/NavigationViewModel.java

```diff
@@ -42,8 +42,9 @@
 import com.mapbox.navigation.ui.camera.DynamicCamera;
 import com.mapbox.navigation.ui.feedback.FeedbackItem;
 import com.mapbox.navigation.ui.instruction.BannerInstructionModel;
-import com.mapbox.navigation.ui.instruction.InstructionModel;
-import com.mapbox.navigation.ui.junction.RouteJunctionModel;
+import com.mapbox.navigation.ui.instruction.internal.InstructionModel;
+import com.mapbox.navigation.ui.internal.ConnectivityStatusProvider;
+import com.mapbox.navigation.ui.internal.junction.RouteJunctionModel;
 import com.mapbox.navigation.ui.summary.SummaryModel;
 import com.mapbox.navigation.ui.voice.NavigationSpeechPlayer;
 import com.mapbox.navigation.ui.voice.SpeechPlayer;
```

Comment:
Why most of `NavigationViewModelTest`s are commented out 🤔?

Reply:
Will be taken care separately in a different ticket.

Reply:
Tracking here https://github.com/mapbox/mapbox-navigation-android/pull/3104 👍